### ISSUE TITLE
Toolbar buttons don't appear

### ISSUE THREAD

**Issue by Divide-By-0** [jitsi/jitsi-meet#8945]
### Description:
The toolbar buttons do not appear, and the view changes from tile view to current speaker view with no way to change it back.

### Steps to reproduce:

1. In tile view, click to change to current speaker view
2. No way to change back, or click again to enter tile view.

### Expected behavior:

1. In tile view, clicking should not change view.
2. If clicking does change view to active speaker view, a click again should enter tile view.

### Actual behavior:

1. In tile view, click to change to current speaker view
2. No way to change back, and clicking again stays in active speaker mode. 

### Server information:

- Jitsi Meet version: Default version.
- Operating System: Chrome embed.

### Client information:

- Browser / app version: Chrome embed.
- Operating System: MacOS.

### Additional information:
Tile view toggle:
```
            // Hack the tile view. Runs only once per user. If they clear their cookies, then it will have the opposite effect and disable their tile view.
            if (!tileViewEnabled) {
              console.error("Going to toggle the tile view");
              jitsi.executeCommand("toggleTileView");
              tileViewEnabled = false;
              window.localStorage.setItem(
                "tileViewEnabled",
                JSON.stringify(true)
              );
            }
```
Config file:
```
export const optionsConfig = {
  interfaceConfigOverwrite: {
    SHOW_JITSI_WATERMARK: false,
    SHOW_WATERMARK_FOR_GUESTS: false,
    SHOW_CHROME_EXTENSION_BANNER: false,
    TOOLBAR_BUTTONS: ["microphone", "camera", "tileview"],
    VERTICAL_FILMSTRIP: true,
    SETTINGS_SECTIONS: ["devices"],
    filmStripOnly: false,
    FILM_STRIP_MAX_HEIGHT: 80,
    DEFAULT_BACKGROUND: "transparent",
    JITSI_WATERMARK_LINK: "null",
    MOBILE_APP_PROMO: false,
    DISABLE_FOCUS_INDICATOR: true,
    DISABLE_DOMINANT_SPEAKER_INDICATOR: true,
    DISPLAY_WELCOME_PAGE_CONTENT: false,
    TILE_VIEW_MAX_COLUMNS: 1,
    DEFAULT_REMOTE_DISPLAY_NAME: "Name",
    DISABLE_VIDEO_BACKGROUND: false,
    VIDEO_QUALITY_LABEL_DISABLED: true,
    CONNECTION_INDICATOR_DISABLED: true,
    HIDE_KICK_BUTTON_FOR_GUESTS: true,
  },
  configOverwrite: {
    enableNoisyMicDetection: false,
    enableTalkWhileMuted: false,
    enableNoAudioDetection: false,
    enableWelcomePage: false,
    prejoinPageEnabled: false,
    resolution: 480,
  },
};
```

**Comment by saghul** [comment]
What deployment are you targeting? We recently changed the tile view button so it will now also be shown for a single user, it used to be hidden unless there where 2 participants in the room.

**Comment by Divide-By-0** [comment]
Not sure, how do you change deployment ? It's now the case that it no longer shows the button in single speaker mode all the time.

**Comment by saghul** [comment]
What domain are you passing when creating the JitsiMeetExternalAPI object?

**Comment by Divide-By-0** [comment]
We do `JitsiMeetExternalAPI("meet.jit.si", options);`. Would it help to share the full options?

**Comment by saghul** [comment]
Please share the full sample. You have that bit of custom code too, I wonder if it messes something up.

**Comment by Divide-By-0** [comment]
The options code takes the options above and does this:
```
  const options = Object.assign(
    {
      roomName,
      parentNode: "jitsi-container",
    },
    optionsConfig
  );
```

The toggle hack only applies when someone joins, so I don't think it's applicable.

**Comment by Divide-By-0** [comment]
Can you reproduce that clicking active speaker view does not return the video chat to tile view?

**Comment by Divide-By-0** [comment]
Is there a way to customize those buttons so we can convert between 1 and 2 columns in tile view too? I.e. editing config on the fly?

**Comment by saghul** [comment]
I'm not 100% sure what problem you are reporting really. Here is what I tried:

- a simple example with the same options as yourself
- the tile view button is displayed and works
- clicking on any tile switches back to stage view
- clicking on the tile view button switches back to tile view

Now, since you forced a single column, tile view and stage view look very similar and you need to scroll down to see all tiles, is that really what you want?

I think it'd be better if you describe what you are trying to accomplish.

**Comment by Divide-By-0** [comment]
Yes! In stage view, i want to be able to click any tile and switch back to tile view.

The videos are displayed in a column of width

**Comment by saghul** [comment]
In stage view clicking on a video switches that video to be "on stage" ie, be displayed large. This behavior is not configurable.

**Comment by Divide-By-0** [comment]
I see, is there a way we can modify the jitsi source code to change this? It seems strange you can only go one way.

**Comment by saghul** [comment]
> I see, is there a way we can modify the jitsi source code to change this?

Sure: https://github.com/jitsi/jitsi-meet/blob/e803e8cfd9f87ed4ac02098de74da9f0c5457ff7/react/features/filmstrip/components/web/Thumbnail.js#L516

> It seems strange you can only go one way from tile > stage, but not back by clicking.

If you click on a participant you "focus" on them bu putting them on the large view, you wouldn't be able to do this with the change you are suggesting.

I'm closing this, since there is nothing actionable. You can modify the line I pointed out above to toggle tile view instead.
